webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/BreadcrumbSwitcherPopover.js

```diff
@@ -47,12 +47,15 @@ const BreadcrumbSwitcherPopover = ({
     const createItemProps = item => {
       const { id, url, name } = item;
       const key = `${id}-${name}`;
-
+      const urlWithName = url ? url.replace(id, name) : '';
       const itemProps = {
         key,
         id: key,
         className: 'no-border',
-        active: url === window.location.pathname,
+        active:
+          url === window.location.pathname ||
+          urlWithName === window.location.pathname ||
+          urlWithName === `${window.location.pathname}/`,
```

Comment:
What if the `window.location.pathname` ends with slash? It feels like too many variations we should support.

What do you think about:
```jsx
// can be shared globaly
const removeLastSlashFromPath = path => path.replace(/^\/+|\/+$/g, '');

const pathname = removeLastSlashFromPath(window.location.pathname);
const url = item.url ? removeLastSlashFromPath(item.url) : '';

```

Reply:
Removing slashes from the url var causes that click on the item to redirect to 'hosts/hosts/:id'. I will use the function in the condition